src/components/textbox/textbox.pw.tsx

```diff
@@ -0,0 +1,1125 @@
+import { expect, test } from "@playwright/experimental-ct-react17";
+import { TextboxProps } from "components/textbox";
+import React from "react";
+import { HooksConfig } from "../../../playwright";
+import {
+  getDesignTokensByCssProperty,
+  checkAccessibility,
+  checkGoldenOutline,
+  verifyRequiredAsteriskForLabel,
+} from "../../../playwright/support/helper";
+import {
+  fieldHelpPreview,
+  getDataElementByValue,
+  getDataComponentByValue,
+  getDataRoleByValue,
+  tooltipPreview,
+  characterLimit,
+} from "../../../playwright/components";
+import {
+  TextboxComponent,
+  TextboxComponentInputRef,
+  TextboxComponentWithLeftChildren,
+  TextboxComponentWithPositionedChildren,
+  TextboxValidationsAsAStringWithTooltipDefault,
+  TextboxValidationsAsABoolean,
+  TextboxValidationsAsAString,
+  TextboxValidationsAsAStringWithTooltipCustom,
+  TextboxValidationsAsAStringDisplayedOnLabel,
+  TextboxNewDesignsValidation,
+} from "./components.test-pw";
+import {
+  textbox,
+  textboxInput,
+  textboxPrefix,
+} from "../../../playwright/components/textbox";
+
+import {
+  CHARACTERS,
+  SIZE,
+  VALIDATION,
+} from "../../../playwright/support/constants";
+
+import { BUTTON, ICON } from "../../../playwright/components/locators";
+import Box from "../../../src/components/box";
+
+const testData = [CHARACTERS.DIACRITICS, CHARACTERS.SPECIALCHARACTERS];
+
+const keysToTrigger = ["Enter", "Space"] as const;
+
+test.describe("Prop checks for Textbox component", () => {
+  ([
+    [SIZE.SMALL, "32px", "--sizing400"],
+    [SIZE.MEDIUM, "40px", "--sizing500"],
+    [SIZE.LARGE, "48px", "--sizing600"],
+  ] as [TextboxProps["size"], string, string][]).forEach(
+    ([size, height, token]) => {
+      test(`should render with ${size} size and ${height} height when size prop is ${size}`, async ({
+        mount,
+        page,
+      }) => {
+        await mount(<TextboxComponent size={size} />);
+
+        await expect(textbox(page)).toHaveCSS("min-height", height);
+
+        const tokenValues = await getDesignTokensByCssProperty(
+          page,
+          textbox(page),
+          "min-height"
+        );
+        expect(tokenValues[0]).toContain(token);
```

Comment:
I can't quite remember how the util formats it, I think it strips out everything but the token name. But regardless, we should check for an exact match here, otherwise we don't know it's right - same as we do in the other token tests. That means checking against something like `(${token})` if the output does contain brackets, or whatever.

Reply:
Yeah ignore me, `toBe` works fine